website/src/contribute/index.md

```diff
@@ -172,6 +174,9 @@ To contribute code, you need
    Use `R: @username` in the pull request to notify a reviewer.
 1. If you don't get any response in 3 business days, email the dev@ list to ask for someone to look at your pull
    request.
+1. Review feedback typically leads to follow-up changes. Please add these changes as additional "fixup" commits to the
```

Comment:
I don't think we agreed on this, did we? This used to be critical for GitHub to not lose comments, but I think that is no longer the case. I see a lot of PRs that don't do this and it seems OK. In fact, when a large PR is reviewed incrementally like this I think it tends to let in worse changes because the increments after the first round look OK even though the resulting code is not so good.

FWIW reviewable.io does not require this and has its own incremental review that we have toyed with. Anyone can choose to use it on a per-review basis.

Reply:
I don't think it was part of the discussion, rather it seems to be the undocumented status quo from looking at what happens currently (changes being added incrementally to PRs)? I don't have a strong preference one way or the other.

Reply:
IIRC, github comments are tied to diffs, so they can be lost if the code changes too much and the commits are gone. I personally like seeing just the changes, as it gives me the choice of viewing everything vs. the incremental diff (and the default is still everything). If consensus isn't obvious here, maybe back to the list?

Reply:
Changed wording.